pkgs/by-name/br/bruno/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  dpkg,
+  makeWrapper,
+  wrapGAppsHook,
+  alsa-lib,
+  gtk3,
+  mesa,
+  nspr,
+  nss,
+  systemd,
+  ...
+}:
+stdenv.mkDerivation rec {
+  pname = "bruno";
+  version = "0.16.5";
+
+  src = fetchurl {
+    url = "https://github.com/usebruno/bruno/releases/download/v${version}/bruno_${version}_amd64_linux.deb";
+    hash = "sha256-VULbXA08okJdX8ItBa4eukENw2zZM2iObbnOXvXhXnY=";
+  };
+
+  nativeBuildInputs = [autoPatchelfHook dpkg makeWrapper wrapGAppsHook];
+
+  dontWrapGApps = true;
+
+  buildInputs = [
+    alsa-lib
+    gtk3
+    mesa
+    nspr
+    nss
+  ];
+
+  runtimeDependencies = [(lib.getLib systemd)];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p "$out/bin"
+    cp -R opt $out
+    cp -R "usr/share" "$out/share"
+    chmod -R g-w "$out"
+    runHook postInstall
+  '';
+
+  postFixup = ''
+    wrapGApp "$out/opt/Bruno/bruno"
+    ln -s "$out/opt/Bruno/bruno" "$out/bin/bruno"
+    substituteInPlace "$out/share/applications/bruno.desktop" \
+      --replace "/opt/Bruno/bruno" "$out/bin/bruno"
+  '';
+
```

Comment:
BTW because of the new update please test using `nix-update-script` with passthru.updateScript and test if it updates properly so in the future bots like r-ryantm can do most of the work.

There is more details searching for `passthru.updateScript` in the [nixpkgs manual](https://nixos.org/manual/nixpkgs/unstable/)